lib/css_parser/rule_set.rb

```diff
@@ -113,8 +113,44 @@ def delete(property)
       end
       alias remove_declaration! delete
 
+      def replace_declaration!(property, replacements)
```

Comment:
I was expecting this to be something like:
```ruby
declarations.each_with_object({}) do |(k, v), h|
  if k == property
    h.merge! replacements
  else
    h[k] = v
  end
end
```

Reply:
We should obey `!important` & declarations order, because we shouldn't override subsequent declarations and declarations before with `!important` if we're not, so I don't see easier way to do that. Will add `.key?` check for the replaced property though.